### ISSUE TITLE
lottie: expressions error, elements appear upside down

### ISSUE THREAD

**Issue by theashraf** [thorvg/thorvg#2724]
Hello team
some elements in the provided lottie are being rendered upside down (rotated 180deg), any ideas ? 

lottie file -> [reportLoadingDone.json](https://github.com/user-attachments/files/16974228/reportLoadingDone.json)
thorvg -> v0.14.8
related issue -> https://github.com/LottieFiles/dotlottie-web/issues/351

**Comment by hermet** [comment]
seems related to expressions.

**Comment by hermet** [comment]
curlprit:
```
"var $bm_rt;\nvar $bm_rt;\nvar amp, freq, decay, n, n, t, t, v;\namp = 0.06;\nfreq = 2;\ndecay = 4;\n$bm_rt = $bm_rt = n = 0;\nif (numKeys > 0) {\n    $bm_rt = $bm_rt = n = nearestKey(time).index;\n    if (key(n).time > time) {\n        n--;\n    }\n}\nif (n == 0) {\n    $bm_rt = $bm_rt = t = 0;\n} else {\n    $bm_rt = $bm_rt = t = sub(time, key(n).time);\n}\nif (n > 0) {\n    v = velocityAtTime(sub(key(n).time, div(thisComp.frameDuration, 10)));\n    $bm_rt = $bm_rt = add(value, div(mul(mul(v, amp), Math.sin(mul(mul(mul(freq, t), 2), Math.PI))), Math.exp(mul(decay, t))));\n} else {\n    $bm_rt = $bm_rt = value;\n}"
```
[sample.json](https://github.com/user-attachments/files/16989654/sample.json)

**Comment by mgrudzinska** [comment]
@hermet 

1. from https://helpx.adobe.com/after-effects/using/expression-language-reference.html:

```
key(index)
Return type: Key or MarkerKey.

Argument type: index is a Number.

Returns the Key or MarkerKey object by number. For example, key(1) returns the first keyframe.
```
 we count frames from 0, but in expressions the first index should be 1 (maybe it is enough in _key() decrease the key value and in _nearestKey() increase the index value?)
 
 also support for floats in _velocityAtTime is missing

**Comment by hermet** [comment]
@mgrudzinska yes, I know the issue details!

**Comment by hermet** [comment]
the fix will be in v0.15
